services/event/db/schema.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose')
+const Joigoose = require('joigoose')(mongoose)
+const Joi = require('joi')
+
+const joiEventSchema = Joi.object({
+  dbType: Joi.string().required(),
+  name: Joi.string().required().meta({ unique: true }),
+  dbURL: Joi.string().required(),
+  regCount: Joi.number().required().meta({ default: 0 }),
+  date: Joi.string().required()
```

Comment:
Remove this too, it's irrelevant to show the date of the event here, only the placeholder micro service needs to deal with dates

Reply:
done